src/main/java/com/tstepnik/planner/domain/task/TaskDto.java

```diff
@@ -0,0 +1,40 @@
+package com.tstepnik.planner.domain.task;
+
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.validation.constraints.NotEmpty;
+
+public class TaskDto {
+
+    @NotEmpty
+    private String description;
+
+    @Enumerated(EnumType.STRING)
+    private Importance importance;
+
+    private boolean isDone;
```

Comment:
Instead of `boolean` you can use `Boolean` and it will be set to false by default if object is missing a value. Safer because of NPE (nullpointerexceptions)

Reply:
Ok I will change it but nevertheless this branch has nothing to do with tasks fields.